cmd/argocd-lovely-plugin/main_test.go

```diff
@@ -83,7 +84,7 @@ func matchExpectedWithStore(path string, givenValue string) error {
 	if err != nil {
 		got := path + "/got.txt"
 		err := os.Remove(got)
-		if err != nil {
+		if err != nil && !os.IsNotExist(err) {
```

Comment:
This is an unrelated change, but I found that tests often failed in a less helpful way because this "just for test logging/helping" file didn't exist yet!

Reply:
I think that's fine.